silverstrike/templates/silverstrike/account_detail.html

```diff
@@ -3,139 +3,214 @@
-  </div>
-</div>
-</div>
-</div>
-<div class="box">
-<div class="box-header with-border">
-  <h3 class="box-title">{% trans 'Transaction list' %}</h3>
-</div>
-<div class="box-body">
-<table class="table table-striped">
-  <tr>
-    <th>{% trans 'Title' %}</th>
-    <th>{% trans 'Date' %}</th>
-    <th class="hidden-xs">{% trans 'Booking date' %}</th>
-    <th>{% trans 'Amount' %}</th>
-    <th class="hidden-xs">{% trans 'Account' %}</th>
-    <th class="hidden-xs">{% trans 'Category' %}</th>
-  </tr>
-{% for transaction in transactions %}
-<tr>
-  <td><a href="{% url 'transaction_detail' transaction.transaction.pk %}">{{ transaction.transaction.title }}</a></td>
-  <td>{{ transaction.transaction.date }}</td>
-  <td class="hidden-xs">{{ transaction.date }}</td>
-  <td{% if transaction.amount < 0 %} class="text-red"{% elif transaction.is_deposit %} class="text-green"{% endif %}>{{ transaction.amount }}</td>
-  <td class="hidden-xs">{% if transaction.is_system %}{% trans 'Reconcilation' %}{% else %}<a href="{{ transaction.opposing_account.get_absolute_url }}">{{ transaction.opposing_account }}</a>{% endif %}</td>
-  <td class="hidden-xs">
-    {% if transaction.category %}
-        <a href="{{ transaction.category.get_absolute_url }}">{{ transaction.category }}</a>
-    {% endif %}
-  </td>
-  </tr>
-{% endfor %}
-</table>
-</div>
-</div>
 {% endblock %}
 
 {% block scripts %}
-{% if dstart and dend %}
-<script src="{% static 'vendor/js/Chart.bundle.min.js' %}"></script>
-<script type="text/javascript">
-$.getJSON("{% url 'api_account_balance' account.id dstart dend %}", {}, function(res, status) {
-  var ctx = document.getElementById("myChart").getContext('2d');
-  var myChart = new Chart(ctx, {
-      type: 'line',
-      options: {
-          legend: {
-            position: 'none'
-          },
-          scales: {
-              yAxes: [{
-                  ticks: {
-                      beginAtZero: true
-                  }
-              }]
-          }
-      },
-      data: {
-          labels: res.labels,
-          datasets: [{
-              label: 'Balance',
-              fill: false,
-              pointRadius: 2,
-              borderColor: '#357ca5',
-              backgroundColor: '#357ca5',
-              data: res.data
-          }]
-      }
-    });
-  });
-</script>
-{% else %}
-<script type="text/javascript">
-  document.getElementById("balance-chart").innerHTML = "<b>{% trans 'Not enough data to show a chart' %}</b>";
-</script>
-{% endif %}
+    {% if dstart and dend %}
+        <script src="{% static 'vendor/js/Chart.bundle.min.js' %}"></script>
+        <script type="text/javascript">
+
+            var start_date = moment(new Date("{{ dstart }}"));
+            var end_date = moment(new Date("{{ dend }}"));
+
+            get_account_balance(start_date, end_date);
+
+            $('#daterange').daterangepicker({
+                "ranges": {
+                    "Today": [
```

Comment:
the today range makes no real sense. Most users won't have more than a dozen of transactions per day per account.

I guess things like (assuming the month is august 2018): 
* last week
* last 30 days
* August 2018
* July 2018
* June 2018
* last six month
* last year

are somewhat ok. I guess I will most often use it to get one or two months back.